include/ignition/gazebo/components/BatteryPowerLoad.hh

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2018 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#ifndef IGNITION_GAZEBO_COMPONENTS_BATTERYCONSUMPTION_HH_
+#define IGNITION_GAZEBO_COMPONENTS_BATTERYCONSUMPTION_HH_
+
+#include <string>
+
+#include <ignition/gazebo/components/Factory.hh>
+#include <ignition/gazebo/components/Component.hh>
+#include <ignition/gazebo/config.hh>
+
+namespace ignition
+{
+namespace gazebo
+{
+// Inline bracket to help doxygen filtering.
+inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
+namespace components
+{
+  /// \brief Data structure to hold information about the parent and child links
```

Comment:
Knit: Docstring doesn't convey information.

Reply:
Good catch, fixed in https://github.com/gazebosim/gz-sim/pull/1811/commits/b4c52e103fc90b10f4dda1e0dbce685c63cad717.